client/src/partials/vouchers/simple.js

```diff
@@ -30,8 +30,22 @@ function SimpleJournalVoucherController(AppCache, Vouchers, Accounts, Session, u
     { label : 'VOUCHERS.SIMPLE.TITLE' }
   ];
 
-  // bind the submit method
+  // transfer type
+  vm.transferType = [
```

Comment:
This would be better placed in the `VoucherService`.  Could you put it in the property `Vouchers.transferTypes`?  That way the controller only does the data binding.